projects/Covision.md

```diff
@@ -0,0 +1,58 @@
+## Project Name
+Covision
+
+## Category 
+Application
+
+## Project Description
+Decentralised disease outbreak surveillance system using IPFS to efficiently manage disease reportings.
+
+## Use of IPFS, Filecoin and Libp2p
+Most of the countries need to handle their own database due to lack of a decentralised platform. This leads to complications when global data is being compiled.  being a decentralised platform being handled by behind the scene Smart Contracts and IPFS provides one global platform to report medical data.
+Authenticity and quality of medical data is another issue when we are being faced with pandemics. With the proper use of IPFS technology Covision has been made such that only verified individuals/organisations can write data.
+
+## Project Status
+Under development
+
+## Previews
+![Screenshot 1](https://media.discordapp.net/attachments/873587956013752340/873588008325111868/img1.png)
+![Screenshot 2](https://cdn.discordapp.com/attachments/873587956013752340/873588011340816444/img2.png)
+![Screenshot 3](https://cdn.discordapp.com/attachments/873587956013752340/873588011948970044/img3.png)
+![Screenshot 4](https://cdn.discordapp.com/attachments/873587956013752340/873588013463109702/img4.png)
+![Screenshot 4](https://media.discordapp.net/attachments/873587956013752340/873587990205714462/img5.png)
+
+## Target Audience
+Public , Government .
+
+## Github repo
+https://github.com/Mr-Cryptocoder/Covision
+
+## Rough estimated user base (if applicable)
+6 beta users
+
+## Website
+The website is set to be hosted in the next 30 days
+
+
+## Team Info
+### Team Size
+1
+
+### Team members
+Sanil
+	- Full stack web developer.
+	- Proficient with JS, IPFS , libp2p, Blockchain.
```

Comment:
Minor formatting, please change to `Proficient with JS, IPFS, libp2p, Blockchain.`